.circleci/unittest/linux/scripts/setup_env.sh

```diff
@@ -56,6 +56,8 @@ cat "${this_dir}/environment.yml"
 if [[ $OSTYPE == 'darwin'* ]]; then
   PRIVATE_MUJOCO_GL=glfw
 else
+  # Software rendering requires GLX and OSMesa.
+  yum install -y mesa-libOSMesa-devel.x86_64 mesa-libGL-devel.x86_64 mesa-libGLU-devel.x86_64
```

Comment:
Do we still need to install osmesa via conda after that?
Also, do you think we could make it work with `MUJOCO_GL=egl`? It should speed up some tests by 2 orders of magnitude or so...

Reply:
I can def update and see what happens :)

Reply:
It seems that now egl is failing. 
Sometimes it works when you ssh on the CircleCI instance (see my opened issue on Mujoco, link in the issue description).
They suggested that if it is the case, we should look at the env variable with and without ssh.
I don't know if you have the credentials to ssh on those machines though...
That being said, if tests pass with osmesa now I'm already more than happy! Solving it for egl is really more like going the extra mile

Reply:
haha okay I will revert --> I probably don't have ssh access since this is my first week...

Reply:
The cpu tests are passing just fine -- but none of the gpu tests passed. Any suggestion?

Reply:
That's the core of this issue:
rendering for dm_control is tested with GPUs. It used to work, but now that the osmesa we installed via conda is not accessible, we must find a workaround.
I would suggest to try to ssh onto the machine: click on the tests that fails and then ssh with this
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/25529882/182761011-28e370d9-4c03-497b-ac4b-af20e287d3ae.png">

It's very likely that you won't have the credentials to do that tho...

Reply:
I don't have the write permission to rerun the job with SSH...
I remember in the bug report you mentioned that when logging through SSH the bug disappears. Is it still true in the current PR? Could you help trigger it and test it? TY!

Reply:
@tongbaojia 
I have done a bit of debugging.
With 93fb291468f2a731ae1456afac42ed191f3dd44d, running the `run_tests.sh` as is did not succeed. After a few attemps, I realized that installing 

```
pip3 install pyrender
pip3 install pyopengl --upgrade
```

_in that order_ worked! You must make sure that 
```
export MUJOCO_GL=egl
export PYOPENGL_PLATFORM=egl
```
(they should be properly set in the conda env)

The tests seem to run fine with that, but it's in ssh (might break in pure headless).

Hope that helps!

Reply:
Here's the script of run_tests.sh that i managed to run:
```
#!/usr/bin/env bash

set -e

eval "$(./conda/bin/conda shell.bash hook)"
conda activate ./env

pip3 install pyrender
pip3 install pyopengl --upgrade

export MUJOCO_GL=egl
export PYOPENGL_PLATFORM=egl

export PYTORCH_TEST_WITH_SLOW='1'
python -m torch.utils.collect_env
# Avoid error: "fatal: unsafe repository"
git config --global --add safe.directory '*'

root_dir="$(git rev-parse --show-toplevel)"
env_dir="${root_dir}/env"
lib_dir="${env_dir}/lib"

# solves ImportError: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found
export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$lib_dir
export MKL_THREADING_LAYER=GNU

pytest test/smoke_test.py -v --durations 20
pytest test/smoke_test_deps.py -v --durations 20
python3 test/test_libs.py  # those are the tests that should break
pytest --instafail -v --durations 20
```

I have added 5 lines of code, for debugging. The packages should be installed elsewhere (in setup_env.sh) and the env variable should be set in the conda env. The test/test_lib.py is already run by pytest so we can remove it.

Reply:
@vmoens I tired to just use the run_tests.sh in [a8e52ae](https://github.com/facebookresearch/rl/pull/335/commits/a8e52ae07e2d6adbdb6abe1beecb0e01dd57c839). Unfortunately it doesn't seem to work in headless mode. 

Could it be the ssh process changed some display env variables?